examples/bdd/lifelong_learning_bench/testalgorithms/rfnet/RFNet/myutils/metrics.py

```diff
@@ -0,0 +1,146 @@
+import numpy as np
+
+
+class Evaluator(object):
+    def __init__(self, num_class):
+        self.num_class = num_class
+        self.confusion_matrix = np.zeros((self.num_class,)*2)  # shape:(num_class, num_class)
+
+    def Pixel_Accuracy(self):
+        Acc = np.diag(self.confusion_matrix).sum() / self.confusion_matrix.sum()
+        return Acc
+    
+    def Pixel_Accuracy_Class_Curb(self):
+        Acc = np.diag(self.confusion_matrix) / self.confusion_matrix.sum(axis=1)
+        print('-----------Acc of each classes-----------')
+        print("road         : %.6f" % (Acc[0] * 100.0), "%\t")
+        print("sidewalk     : %.6f" % (Acc[1] * 100.0), "%\t")
+        Acc = np.nanmean(Acc[:2])
+        return Acc
+        
+
+    def Pixel_Accuracy_Class(self):
+        Acc = np.diag(self.confusion_matrix) / self.confusion_matrix.sum(axis=1)
+        print('-----------Acc of each classes-----------')
+        print("road         : %.6f" % (Acc[0] * 100.0), "%\t")
+        print("sidewalk     : %.6f" % (Acc[1] * 100.0), "%\t")
+        print("building     : %.6f" % (Acc[2] * 100.0), "%\t")
+        print("wall         : %.6f" % (Acc[3] * 100.0), "%\t")
+        print("fence        : %.6f" % (Acc[4] * 100.0), "%\t")
+        print("pole         : %.6f" % (Acc[5] * 100.0), "%\t")
+        print("traffic light: %.6f" % (Acc[6] * 100.0), "%\t")
+        print("traffic sign : %.6f" % (Acc[7] * 100.0), "%\t")
+        print("vegetation   : %.6f" % (Acc[8] * 100.0), "%\t")
+        print("terrain      : %.6f" % (Acc[9] * 100.0), "%\t")
+        print("sky          : %.6f" % (Acc[10] * 100.0), "%\t")
+        print("person       : %.6f" % (Acc[11] * 100.0), "%\t")
+        print("rider        : %.6f" % (Acc[12] * 100.0), "%\t")
+        print("car          : %.6f" % (Acc[13] * 100.0), "%\t")
+        print("truck        : %.6f" % (Acc[14] * 100.0), "%\t")
+        print("bus          : %.6f" % (Acc[15] * 100.0), "%\t")
+        print("train        : %.6f" % (Acc[16] * 100.0), "%\t")
+        print("motorcycle   : %.6f" % (Acc[17] * 100.0), "%\t")
+        print("bicycle      : %.6f" % (Acc[18] * 100.0), "%\t")
+        print("dynamic      : %.6f" % (Acc[19] * 100.0), "%\t")
+        print("stair        : %.6f" % (Acc[20] * 100.0), "%\t")
```

Comment:
For commercial services like sedna and kubeedge, use the log to output to screen instead of using print, which is not safe.